docs/guide/use_cases/nlb_tls_termination/index.md

```diff
@@ -0,0 +1,127 @@
+---
+title: TLS Termination with Network Load Balancer
+---
+
+## Motivation
+
+![diagram illustrating connection between network load balancer and cluster](fig.jpg)
+
+Managing TLS certificates (and related configuration) for production cluster workloads is both time consuming, and high risk. For example, storing multiple copies of a certificate secret key in the cluster may increases the chances of it being compromised. Additionally, TLS can be complicated to configure and implement properly. 
+
+Traditionally, TLS termination at the load balancer step required using more expensive application load balancers (ALBs). AWS introduced TLS termination for network load balancers (NLBs) for enhanced security and cost effectiveness. 
+
+The TLS implementation used by the AWS NLB is formally verified and maintained. Additionally, AWS Certificate Manager (ACM) is used, fully isolating your cluster from access to the private key. 
+
+## Solution Overview
+
+An external client transmits a request to the NLB. The request is encrypted with TLS using the production (e.g., client facing) certificate, and on port 443. 
+
+The NLB decrypts the request, and transmits it on to your cluster on port 80. It follows the standard request routing configured within the cluster. Notably, the request received within the cluster includes the actual origin IP address of the external client. 
+
+Alternate ports may be configured. End-to-end encryption technically requires the segment between the NLB and cluster pods be encrypted also. A follow-up post will describe the NLB originating TLS based on a cluster certificate. 
+
+!!! note
+    The NLB may be configured to maintain the source (i.e., client) IP address. However, there are some limitations. Review [Client IP Preservation](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/load-balancer-target-groups.html#client-ip-preservation) in the AWS docs. 
+
+For fig.jpg, the "Source IP Address is Maintained" note is not always true. It depends on the target group attributes from the pov of the NLB, and the spec.externalTrafficPolicy for the pod in case of instance target.
```

Comment:
fig.jpg no longer has the note, we need to remove this line.

Reply:
Good catch thank you 😅